Makefile

```diff
@@ -1,3 +1,5 @@
+QGISDIR=.local/share/QGIS/QGIS3/profiles/jhemmi
```

Comment:
Is everything new in the Makefile running?
I couldn't use the `zip` option (the only one that I tested).

If the new options aren't working, please remove them, which means, let's keep the Makefile untouched.

Reply:
Makefile out from repo